src/Import/include/FMI3/fmi3_import_convenience.h

```diff
@@ -180,6 +181,30 @@ void fmi3_default_callback_logger(fmi3_instance_environment_t inst, fmi3_string_
 FMILIB_EXPORT
 void fmi3_import_init_logger(jm_callbacks* cb, fmi3_logger_context_t* loggerCallbacks);
 
+/**
+    \brief Get variable description by variable name. Alias variable names result in the description of the alias variable.
+    @param fmu - An fmu object as returned by fmi3_import_parse_xml().
+    @param name - variable name
+    @return Description, empty string "" if missing description. NULL if no variable with given name exists.
+*/
+FMILIB_EXPORT
+const char* fmi3_import_get_variable_description_by_name(fmi3_import_t* fmu, const char* name);
+
+/**
+    \brief Get variable display unit by variable name. Alias variable names result in the display unit of the alias variable.
+    @param fmu - An fmu object as returned by fmi3_import_parse_xml().
+    @param name - variable name
+    @return "display unit" object if existant. NULL if no variable exists, or no display unit.
```

Comment:
Actually having second thoughts on this one, due to the uncertainty with NULL returns.